definitions/variable/technology/power-plant.yaml

```diff
@@ -188,69 +176,40 @@
 - Maximum Energy Charge|Electricity|Energy Storage System:
     description: This is the upper bound that energy storage system could charge energy
       at time "t".
-    unit:
-    - MWh
-    - GWh
+    unit: GWh
 - Initial Storage Level|Electricity|Energy Storage System:
     description: Initial energy stored in the energy storage system unit at first
       load level. (ESS that includes hydro, battery, etc.)
```

Comment:
```suggestion
```
I suggest to remove this variable:
1/ this file is meant for techno characteristics of units, which this is not
2/ initial storage with no  date is very ambiguous and in electricity-opeation we already have the variable Storage level (or sth like that) which for the date corresponding to what is 'initial' will have the same content
3/ initial is a case study parameter.....
(i think this was a variable I created so sorry.....)